index.html

```diff
@@ -1,54 +1,277 @@
+            </ul>
+          </div>
+          <div class="see-project-btn">
+            <a href="#" class="get-resume-link">Get My Resume</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-wrapper">
+      <div class="text">
+        <h2>About <br />Myself</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+      </div>
+      <div class="social-media">
+        <div class="icon-a">
+          <h5><a href="#">Let's connect</a></h5>
+        </div>
         <ul>
-            <li class="html"><a href="#">html</a></li>
-            <li class="css"><a href="#">css</a></li>
-            <li class="js"><a href="#">javascript</a></li>
-            
+          <li>
+            <a href="#">
+              <img src="assets/images/twitter.svg" alt="twitter image" class="twitter" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="assets/images/linkedin.svg" alt="linked in  image" class="linkedin" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="assets/images/m.svg" alt="meduim in  image" class="meduim" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="assets/images/github.svg" alt="github in  image" class="github" />
+            </a>
+
+          </li>
+          <li>
+            <a href="#">
+              <img src="assets/images/angleList.svg" alt="angle list in  image" class="anglelist" />
+            </a>
+          </li>
         </ul>
-        
-    </div>
-    <div class="see-project-btn-stories">
-        
-            <a href="#">See Projects</a>
-    </div>
-    </div>
-</div> 
-</div>
-</div>
+      </div>
+      
+
+      <ul class="list-container">
+        <li class="list">
+          <div class="list-title">
+            <div class="list-icon" id="square-45"></div>
+            <h3>Language</h3>
+          </div>
+             <ul class="list-item">
+            <li><img src="/assets/images/jjs.png" alt="javascript">
+              Javascript</li>
+            <li><img src="/assets/images/hhtml.png" alt="javascript">
+              HTML</li>
+            <li><img src="/assets/images/ccs.png" alt="javascript">
+              CSS</li>
+            </ul>
+        </li>
+
+        <li class="list">
+          <div class="list-title">
+            <div class="list-icon"></div>
+            <h3>Frameworks</h3>
+          </div>
+          
+        </li>
+
+        <li class="list">
+          <div class="list-title">
+            <div class="list-icon" id="circle"></div>
+            <h3>Skills</h3>
+          </div>
+         
+        </li>
+      </ul>
```

Comment:
- [ ****OPTIONAL**** ] Good job again for adding some changes required by the previous reviewer. But I just noticed that while doing those changes you broke again your styling on the `About me` section. I suggest you make the design stick to the one from the Figma template by using the correct `colors` for all the text of the section and making the paragraph **"Hello I’m a software developer! I can he...."** and with a good style.
- [ **OPTIONAL** ] Kindly make sure you also added everything in the **About M..**  section. I just notice that on your **Languages** title you miss a `Chevron-down` icon and the same on `Skills`, `Frameworks` the `chevron-right` icons.
### Don't care about the green border in the figma image
#### Figma design 
![ab](https://user-images.githubusercontent.com/83042742/172007040-8f1c9122-d5e4-4c81-9a67-8b6aa8635266.png) 
#### Your  Design 
![abs](https://user-images.githubusercontent.com/83042742/172007113-f2aa060f-9af0-466b-b6b3-238601cf8839.png) |